Check stderr first before stdout on VCS Install (#9234)
pypa/pip: news/8876.bugfix.rst

> Read stderr output before stdout on VCS install to avoid process freeze with Git

Reviewer: ```suggestion
Detect both stdout and stderr on VCS install VCS install to avoid process freeze with Git.
```
